include/deal.II/grid/tria_accessor.h

```diff
@@ -3630,6 +3630,27 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
   void
   clear_refine_flag() const;
 
+  /**
+   * Return the @p IsotropicRefinementChoices this cell was flagged to be refined
+   * with.
+   */
+  std::uint8_t
+  refine_choice_set() const;
```

Comment:
Even though this name mirrors what we have for `refine_flag_set`, I would still like to hear opinions on this one, because it does not match what we do in most other places of the library, where
```suggestion
  refine_choice() const;
```
would be the canonical choice, see as an example this one: https://github.com/dealii/dealii/blob/e9eb5ab491aab6b0e57e9b552a4e5d64e20077a6/include/deal.II/grid/tria_accessor.h#L700-L707

Reply:
Changed it, as now it also follows `refinement_case()`.